### DISCUSSION TITLE
Check if the location service is active after starting recording

### DISCUSSION THREAD

**Discussion by fvaleri** [osmandapp/OsmAnd#20387] [General]
Hi, I'm using OsmAnd 4.7.17, and I find the trip recording plugin quite convenient. 

It would be good to check if the location service is active after starting recording, and raise an error if not. Wdyt?

**Comment by sjvudp** [comment]
> Hi, I'm using OsmAnd 4.7.17, and I find the trip recording plugin quite convenient.
> 
> It would be good to check if the location service is active after starting recording, and raise an error if not. Wdyt?

How does it relate to the title saying "Trip recording speed unit"?

**Reply by fvaleri** [reply]
I simply forgot to update the title. Let me fix it.
